chrome/views/background.pug

```diff
@@ -2,4 +2,4 @@ doctype html
 
 html
   head
-    script(src=env == 'prod' ? '/background.bundle.js' : 'https://localhost:3000/background.bundle.js')
```

Comment:
chrome://flags/#allow-insecure-localhost

Reply:
I'll remove this file from the PR